### DISCUSSION TITLE
Compile nb.go on windows x64

### DISCUSSION THREAD

**Discussion by ivishalgandhi** [xwmx/nb#335] [General]
Please share steps to compile nb.go on windows x64 

go version go1.22.4 windows/amd64

C:\foss\nb\nb.go>go build  
go: downloading golang.org/x/sys v0.0.0-20201119102817-f84b799fce68
# nb.go
.\main.go:150:51: undefined: unix.Access
.\main.go:150:74: undefined: unix.W_OK
.\main.go:450:23: undefined: syscall.ForkExec

**Comment by boldandbusted** [comment]
I know this post is ancient, but just want to say that I think the Go version of `nb` isn't happening. It has apparently been 5 years since the last update to the Go implementation, and I think for most folks, the shell version of `nb` works fine. :) Cheers.
